capsules/extra/src/ieee802154/README.md

```diff
@@ -0,0 +1,53 @@
+IEEE 802.15.4 Stack
+===================
+
+Tock supports two different implementations of an IEEE 802.15.4 stack in the
+kernel. The first version implements packet framing and a MAC layer, virtualizes
+the 15.4 interface, and provides a userspace interface. The second version
+provides a much more raw version and only provides userspace with direct access
+to the 15.4 radio.
```

Comment:
I think it would be worthwhile to include a comment mentioning how the two stacks are mutually exclusive.

Reply:
I think "different" does the job here. It's also conceivable a board would have both (e.g. if there are two different radios).